AmplifyPlugins/Core/AWSPluginsCore/Model/Decorator/AuthRuleDecorator.swift

```diff
@@ -30,9 +30,17 @@ public enum AuthRuleDecoratorInput {
 public struct AuthRuleDecorator: ModelBasedGraphQLDocumentDecorator {
```

Comment:
Ignore these changes as are part of a different [PR](https://github.com/aws-amplify/amplify-ios/pull/1553), I'll rebase once approved